packages/main/src/Tokenizer.ts

```diff
@@ -172,28 +172,23 @@ class Tokenizer extends UI5Element {
 		this._resizeHandler = this._handleResize.bind(this);
 
 		this._itemNav = new ItemNavigation(this, {
-			currentIndex: "-1",
+			currentIndex: -1,
 			getItemsCallback: this._getVisibleTokens.bind(this),
 		});
 
 		this._scrollEnablement = new ScrollEnablement(this);
 	}
 
-	async onBeforeRendering() {
-		if (this.showPopover && !this._getTokens().length) {
-			const popover = await this.getPopover();
-			popover.close();
-		}
-
+	onBeforeRendering() {
 		this._nMoreCount = this.overflownTokens.length;
 	}
 
 	onEnterDOM() {
-		ResizeHandler.register(this.shadowRoot.querySelector(".ui5-tokenizer--content"), this._resizeHandler);
+		ResizeHandler.register(this.shadowRoot!.querySelector<HTMLElement>(".ui5-tokenizer--content")!, this._resizeHandler);
```

Comment:
this could be replace with contentDom. also if there is no conditional rendering add ! at getter instead of where it is used.